### PR TITLE
Increase CI coverage

### PR DESCRIPTION
**Pull request by hlopko**
I realized we don't run tests on Windows. This PR fixes that. While there I also enabled tests on RBE.

### CODE REVIEW

**Comment by hlopko** [comment]
CC @UebelAndre

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -53,6 +54,14 @@ tasks:
     - "--enable_runfiles" # this is not enabled by default on windows and is necessary for the cargo build scripts
     windows_targets: &windows_targets
     - "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
+    - "..."
+    - "-//bindgen/..."
+    - "-//test/load_arbitrary_tool/..."
+    - "-//test/test_env/..."
+    - "-//test/proto/..."
+    - "-//tools/rust_analyzer/..."
+    - "-//tools/runfiles/..."
```
Comment (illicitonion) [inline .bazelci/presubmit.yml:63]: I think runfiles is meant to work on Windows? (Not sure about the others). But also, definitely let's merge and try to shrink this list over time :)

**Review by illicitonion — APPROVED** [review]

File: test/BUILD [hunk 2]
```diff
@@ -9,9 +9,17 @@ rule_test(
     rule = "@examples//hello_lib:hello_lib",
 )
 
+config_setting(
```
Comment (UebelAndre) [inline test/BUILD:12]: IS there a reason you can't user `//rust/platforms:windows`?

**Review by UebelAndre — CHANGES_REQUESTED** [review]
Sounds good. I'm curious how you actually ensure you're trying to enable more. If there's an automated way to remind you you're not testing everything or try to enable tests and see if something passes based on some change.

File: test/BUILD (hunk 2 above)
Comment (hlopko) [reply test/BUILD:12]: There is not, fixed. Thank you!

File: .bazelci/presubmit.yml (hunk 1 above)
Comment (hlopko) [reply .bazelci/presubmit.yml:63]: Yup, I'll file issues for those.

**Comment by hlopko** [comment]
> Sounds good. I'm curious how you actually ensure you're trying to enable more. If there's an automated way to remind you you're not testing everything or try to enable tests and see if something passes based on some change.

I don't think they will be fixed accidentally, so a system automatically checking if they pass won't be very useful. I'll create issues for them and we should try to fix them soon.

**Pull request merged**